### ISSUE TITLE
Add image embedding models in `models.py`

### ISSUE THREAD

**Issue by sandcobainer** [torchopenl3/torchopenl3#56]
I don't see a `torchopenl3.models.load_image_embedding_model()` function in `models.py`. The original tensor flow implementation has this function in their repo.  

Is this on the roadmap?

**Comment by turian** [comment]
@sandcobainer No, it's not on the roadmap. However, if you are interested in implementing it, we are happy to give you advice on how to do the port.
